charmdet/SciFiHit.cxx

```diff
@@ -22,21 +22,21 @@ void SciFiHit::GetSciFiXYZ(TVector3 &v, int detID)
   double scifiboxdz = ((TGeoBBox*) scifibox->GetShape())->GetDZ(); //semithickness
   
   int32_t partitionID;
-  unsigned int ch;
-  unsigned int boardId;
+  int ch;
+  int boardId;
 
   boardId = (fDetectorID) / pow(10,5);
   ch = (fDetectorID - boardId * pow(10,5));
```

Comment:
Can you merge together the declaration and the initialization of the variable?

like:

``` c++
int ch = (fDetectorID - boardId * pow(10,5));
int boardId = (fDetectorID - boardId * pow(10,5));
```

Also, maybe those can be const?

``` c++
const int ch = (fDetectorID - boardId * pow(10,5));
const int boardId = (fDetectorID - boardId * pow(10,5));
```